src/tests/pools/D3MCompoundPool.t.sol

```diff
@@ -218,35 +217,39 @@ contract D3MCompoundPoolTest is D3MPoolBaseTest {
         assertEq(pool.redeemable(), address(cDai));
     }
 
-    function test_transfer_cdai() public {
-        pool.file("hub", address(this));
+    function test_exit_cdai() public {
```

Comment:
same comment as on Aave, I think we can remove this test and just leave the `caged` one

Reply:
Removed the `cage` on as discussed in the other comment.